src/main/java/homework14/Main.java

```diff
@@ -0,0 +1,49 @@
+package homework14;
+
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.function.Function;
+import java.util.function.ToLongFunction;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Main {
+    private void countWords() {
+        Scanner scanner = new Scanner(System.in);
+        Stream.of(scanner
+                .nextLine()
+                .toLowerCase()
+                .split("[^\\p{L}\\p{Digit}_]+"))
+                .collect(Collectors
+                        .groupingBy(Function.identity(), Collectors.counting()))
+                .entrySet()
+                .stream()
+                .sorted(Map.Entry.comparingByValue())
+                .collect(Collectors
+                        .toMap(Map.Entry
```

Comment:
better:
```suggestion
                        .toMap(Map.Entry::getKey, Map.Entry::getValue, (e1, e2) -> e2, LinkedHashMap::new))
```
or move into new line after " , "

Reply:
thx, done!